qng/common.py

```diff
@@ -0,0 +1,69 @@
+import argparse
```

Comment:
Unused, and same with `json`, `operator`, `random`, `typing.Optional`, and `qng` below.